docs/manuals/man8/pki-server-est.8.md

```diff
@@ -0,0 +1,48 @@
+# pki-server-est 8 "Jul 1, 2022" PKI "PKI EST subsystem management commands"
+
+## NAME
+
+pki-server-est - Command-line interface for managing PKI EST subsystem.
+
+## SYNOPSIS
+
+**pki-server** [*CLI-options*] **est-create** [*command-options*]
+**pki-server** [*CLI-options*] **est-remove** [*command-options*]
+**pki-server** [*CLI-options*] **est-deploy** [*command-options*]
+**pki-server** [*CLI-options*] **est-undeploy** [*command-options*]
+
+## DESCRIPTION
+
+The **pki-server est** commands provide command-line interfaces to manage
+the PKI Enrollment over Secure Transport (EST) subsystem.
+
+**pki-server** [*CLI-options*] **est** [*command-options*]
+    List PKI EST management commands.
+
+**pki-server** [*CLI-options*] **est-create** [*command-options*]
+    Create the initial PKI EST subsystem configuration files in PKI server.
```

Comment:
I had to manually create `backend.conf` and `authorizer.conf`. Shouldn't this command create these files for me?